src/main/java/com/puppycrawl/tools/checkstyle/Utils.java

```diff
@@ -401,4 +403,21 @@ public static int getIntFromField(Field field, Object object) {
             throw new IllegalStateException(exception);
         }
     }
+
+    /**
+     * Closes a stream ignoring IOException.
+     *
+     * @param closeable Closeable object
+     */
+    public static void closeQuietly(Closeable closeable) {
+        if (closeable == null) {
+            return;
+        }
+        try {
+            closeable.close();
+        }
+        catch (IOException e) {
+            // ignore
```

Comment:
ignoring exception is not good, and pmd tells it.
@romani, is there a logger in checkstyle to log this exception?